### PR TITLE
AF-2442: Review Lienzo dependency in Dashbuilder

### PR DESCRIPTION
**Pull request by jesuino**
Review Lienzo dependency in Dashbuilder
--
Merge with:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1215

### CODE REVIEW

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Comment by jesuino** [comment]
jenkins retest this

**Comment by jesuino** [comment]
jenkins retest this

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Comment by mareknovotny** [comment]
Run the FDB only on kiegroup/droolsjbpm-build-bootstrap/pull/1215 here it would be useless if you have parent PR in kie-parent too

File: pom.xml [hunk 1]
```diff
@@ -1241,13 +1231,6 @@
         <version>${version.ch.qos.logback}</version>
       </dependency>
 
-      <!-- Lienzo Charts -->
-      <dependency>
-        <groupId>com.ahome-it</groupId>
-        <artifactId>lienzo-charts</artifactId>
-        <version>${version.com.ahome-it.lienzo-charts}</version>
-      </dependency>
-
```
[the remark this answers is not in the document]
Comment (jesuino) [reply pom.xml]: Hello Marek,

AFAIK the only dependency that used to use this is Dashbuilder Lienzo Renderer, which was removed in this PR and hence why we are also removing this dependency. Kindly let me know if you are aware of any other place that uses it.

Thanks!

[the remark this answers is not in the document]
Comment (mareknovotny) [reply pom.xml]: ok, you are right!

**Comment by mareknovotny** [comment]
Jenkins retest this please

**Pull request merged**